wfrp4e.js

```diff
@@ -77,7 +77,7 @@ import SocketHandlers from "./modules/system/socket-handlers.js";
 /*  Foundry VTT Initialization                  */
 /* -------------------------------------------- */
 
-Hooks.once("init", async function () {
+Hooks.once("init", function () {
```

Comment:
Should have no impact on system itself, since nothing in this callback used `await`.

But I had some issues accessing some things defined here in `init` Hook of my module.